server/utils/utils.ts

```diff
@@ -354,10 +354,14 @@ function postProcessSentenceCase(input: string): string {
       match: /\bN\/A\b/i,
       replace: 'N/A',
     },
+    {
+      match: /\bCategory ([ABCDE])\b/i,
+      replace: (_: string, category: string) => `Category ${category.toUpperCase()}`,
+    },
   ]
   let output = input
   for (const { match, replace } of exceptions) {
-    output = output.replace(match, replace)
+    output = output.replace(match, replace as string)
```

Comment:
ts can't figure out that both options of the `replace` union in the exceptions above are allowed so pretending that it's only ever string :(